src/main/scripts/src/app/core/dto/document.ts

```diff
@@ -20,11 +20,14 @@
 
 export class Document {
 
-  public id: string;
-  public creationDate: string;
-  public createdBy: string;
+  public _id: string;
+  public creation_date: string;
+  public created_by: string;
```

Comment:
We need to have the same structure of front-end as well as back-end DTOs. However, this does not mean it should be the same as database records structure.